charts/dv-pod/values.yaml

```
@@ -0,0 +1,468 @@
+  name: ""
+  # -- Required ClusterRole rules
+  clusterRules:
+    # --  Required to obtain the nodes external IP
+    - apiGroups: [""]
+      resources:
+      - "nodes"
+      verbs:
+      - "get"
+      - "list"
+      - "watch"
+  # -- Required Role rules
+  rules:
+    # -- Required to get information about the serices nodePort.
+    - apiGroups: [""]
+      resources:
+      - "services"
+      verbs:
+      - "get"
+      - "list"
+      - "watch"
+
+# -- Charon service ports
+service:
+  # -- Headless service to create DNS for each statefulset instance
+  clusterIP: None
+  ports:
+    validatorApi:
+      name: validator-api
+      port: 3600
+      protocol: TCP
+      targetPort: 3600
+    p2pTcp:
+      name: p2p-tcp
+      port: 3610
+      protocol: TCP
+      targetPort: 3610
+    monitoring:
+      name: monitoring
+      port: 3620
+      protocol: TCP
+      targetPort: 3620
+
+# -- Pod resources limits and requests
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+# -- Node labels for pod assignment
+## ref: https://kubernetes.io/docs/user-guide/node-selection/
+nodeSelector: {}
+
+# -- Tolerations for pod assignment
+## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+tolerations: {}
+
+# -- Affinity for pod assignment
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+##
+## Example:
+## affinity:
+##   podAntiAffinity:
+##     requiredDuringSchedulingIgnoredDuringExecution:
+##     - labelSelector:
+##         matchExpressions:
+##         - key: app.kubernetes.io/name
+##           operator: In
+##           values:
+##           - charon
+##       topologyKey: kubernetes.io/hostname
+##
+affinity: {}
+
+# -- Used to assign priority to pods
+## ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+priorityClassName: ""
+
+# -- Enable pod disruption budget
+## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb
+podDisruptionBudget:
+  minAvailable: ""
+  enabled: true
+
+# -- allows you to configure and disable automated rolling updates for containers, labels, resource request/limits, and annotations for the Pods in a StatefulSet.
+updateStrategy: RollingUpdate
+
+## Charon command line options
+## ref: https://docs.obol.tech/docs/charon/charon_cli_reference
+config:
+  # -- Comma separated list of one or more beacon node endpoint URLs.
+  beaconNodeEndpoints: "http://localhost:9999" # Placeholder, replace with actual beacon node endpoint
```

```suggestion
  beaconNodeEndpoints: "http://beacon.l1.cluster.svc.local" # Defaults to the Obol Stack
```

This URI doesn't exist yet, so take with a grain of salt. We should align with what we expose on the obol-stack erigon chart.

---

Would be good to have `fallbackBeaconNodeEndpoints:` too in this chart, maybe with publicnode.com as a default, so even the most slapdash installation of this chart might manage to stand up (i.e. in an obol-stack without --mode=full, as an example). (networks will make this tricky though.